modules/ROOT/pages/connections.adoc

```diff
@@ -0,0 +1,2969 @@
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-amazon-rds-postgresql[Amazon RDS PostgreSQL]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-redshift[Amazon Redshift]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-synapse[Azure Synapse]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-databricks[Databricks]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-denodo[Denodo]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-dremio[Dremio]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-genericjdbc[Generic JDBC]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-google-alloydb-postgresql[Google AlloyDB for PostgreSQL]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-gbq[Google BigQuery]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-google-cloud-sql-mysql[Google Cloud SQL for MySQL]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-google-cloud-sql-postgresql[Google Cloud SQL for PostgreSQL]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-google-cloud-sql-sql-server[Google Cloud SQL for SQL Server]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-looker[Looker]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-mode[Mode]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-mysql[MySQL]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-adw[Oracle]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-postgresql[PostgreSQL]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-presto[Presto]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-hana[SAP HANA]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-singlestore[SingleStore]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-snowflake[Snowflake]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-sql-server[SQL Server]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-starburst[Starburst]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-teradata[Teradata]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-trino[Trino]
+
+
+
+This article describes the procedures for setting and managing data connections through the REST APIs. For information about how to set up a data connection in the ThoughtSpot UI, see link:https://cloud-docs.thoughtspot.com/admin/ts-cloud/embrace.html[Data Connections in ThoughtSpot, window=_blank].
+
+== Data Connection APIs
+The connection API endpoints allow you to create, edit, delete, export, and query data connections on a ThoughtSpot cluster. See, the following pages for the REST API endpoints:
+
+* https://developers.thoughtspot.com/docs/connections-api[REST APIs v1, window=_blank]
+* https://developers.thoughtspot.com/docs/rest-apiv2-reference#_connections[REST APIs v2, window=_blank]
+
```

Comment:
Change the first link to "xref:connections-api.adoc[REST API v1].

Reply:
Done.